package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@html-next/vertical-collection",
-  "version": "1.0.0",
+  "version": "2.0.0-0",
```

Comment:
Should we label this as a beta?

Reply:
This format of version number is commonly used by NPM packages to identify a pre-release. The published package on NPM is tagged `next`, which is basically equivalent to `beta` (only the `latest` tag on npm is for stable packages).

You prefer the `-beta` style versioning? Or were you referring to the npm tag?

Reply:
I'm just used to seeing `-beta` on betas.